pallets/vc-management/src/lib.rs

```diff
@@ -82,6 +84,15 @@ pub mod pallet {
 	#[pallet::getter(fn admin)]
 	pub type Admin<T: Config> = StorageValue<_, T::AccountId, OptionQuery>;
 
+	// delegatees who can request (and receive) VCs on users' behalf,
+	// some VCs can only be requested by delegatee accounts (e.g. A13)
+	// delegatees and admins are different:
+	// - admins are meant to manage the pallet state manually, e.g. schema, vcRegistry
+	// - delegatees can request VCs for users, similar to `proxied account`
```

Comment:
Is the delegation restricted to a specific assertion (e.g. Alice can allow Bob to request an `A13` for Alice but that doesn't allow him to request an `A4`)?

If not then it seems to open the door to some big privacy issues (as in, instead of asking Alice for a specific `A6`, Evil Inc. can nudge Alice into giving them a blanket delegation so they can milk _all_ her info) :warning: :thinking:

Reply:
> Is the delegation restricted to a specific assertion (e.g. Alice can allow Bob to request an A13 for Alice but that doesn't allow him to request an A4)?

That's true - delegation only works for specific assertion. As written in the PR description, it's not a generic delegation system. We want it basically for the decoded event where we'd like to request some VC (in this case `A13`) for users on site

Reply:
OK so _only_ `A13` can be requested by a delegatee? e.g. if Daisy is a delegatee she can request an `A13` for Alice but not an `A4` or `A6`? 

(also, is Daisy a delegatee _for Alice_ or _globally_? :thinking:)

Reply:
That's true.

Daisy is a delegatee globally - she can request A13 for any user. This is the feature request we received.